examples/kubernetes/helm/charts/mira/values.yaml

```diff
@@ -6,7 +6,7 @@ image:
   pullPolicy: IfNotPresent
 sideCartContainerName: kubectl
 sideCartContainerArgs: "[ \"proxy\", \"-p\", \"8001\" ]"
-sideCartContainerImage: gcr.io/google_containers/kubectl:v0.18.0-350-gfb3305edcf6c1a
+sideCartContainerImage: lachlanevenson/k8s-kubectl:v1.9.10
```

Comment:
What is this source? Can we just trust it? Nothing more Google official?

Reply:
The google one has not been updated since january 2017.

This is the source: https://github.com/lachie83/k8s-kubectl
It has most stars on github and most pulls on dockerhub from the images I found